source/application/IDEAS/IDEASAppController.cpp

```diff
@@ -156,7 +160,7 @@ void IDEASAppController::setupUserInterface()
 
 	mw_->insertHeading("XRF Detectors", 1);
 
-	ideasKETEKDetailedDetectorViewWithSave_ = new IDEASXRFDetailedDetectorViewWithSave(IDEASBeamline::ideas()->ketek());
+	IDEASKETEKDetailedDetectorView *ideasKETEKDetailedDetectorViewWithSave_ = new IDEASKETEKDetailedDetectorView(IDEASBeamline::ideas()->ketek());
```

Comment:
I also did the same change in Issue1177, to rename `IDEASXRFDetailedDetectorViewWithSave` as `IDEASXRFDetailedDetectorView` :+1: :smile: